src/dnd-list/list.js

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import {getOverride, getOverrideProps} from '../helpers/overrides.js';
+import {
+  Root as StyledRoot,
+  List as StyledList,
+  Item as StyledItem,
+  DragHandle as StyledDragHandle,
+  CloseHandle as StyledCloseHandle,
+  Label as StyledLabel,
+} from './styled-components.js';
+import {List as MovableList} from 'react-movable';
+import Grab from '../icon/grab.js';
+import Delete from '../icon/delete.js';
+
+import type {ListPropsT, SharedStylePropsArgT} from './types.js';
+
+class StatelessList extends React.Component<ListPropsT> {
+  static defaultProps: $Shape<ListPropsT> = {
+    items: [],
+    onChange: () => {},
+  };
+
+  render() {
+    const {overrides = {}, items, onChange, removable} = this.props;
+    const {
+      Root: RootOverride,
+      List: ListOverride,
+      Item: ItemOverride,
+      DragHandle: DragHandleOverride,
+      CloseHandle: CloseHandleOverride,
+      Label: LabelOverride,
+    } = overrides;
+    const Root = getOverride(RootOverride) || StyledRoot;
+    const List = getOverride(ListOverride) || StyledList;
+    const Item = getOverride(ItemOverride) || StyledItem;
+    const DragHandle = getOverride(DragHandleOverride) || StyledDragHandle;
+    const CloseHandle = getOverride(CloseHandleOverride) || StyledCloseHandle;
+    const Label = getOverride(LabelOverride) || StyledLabel;
+    const isRemovable = this.props.removable || false;
+    return (
+      <Root $isRemovable={isRemovable} {...getOverrideProps(RootOverride)}>
```

Comment:
Lines 40-45 can be replaced with `getOverrides` call that provides override component and props to pass as a second element so you won't need to call `getOverrideProps` here:
`const [Root, rootProps] = getOverrides(RootOverride, StyledRoot);`